### PR TITLE
Add Gamepad Support

### PR DESCRIPTION
**Pull request by TheEeveeLovers**
I have programmed and tested directory opening with gamepad and it seems quite alright
This should solve issue #47 (which is already a year old at this point)

### CODE REVIEW

File: imfilebrowser.h [hunk 1]
```diff
@@ -646,6 +646,20 @@ inline void ImGui::FileBrowser::Display()
                     CloseCurrentPopup();
                 }
             }
+            else if (IsKeyPressed(ImGuiKey_GamepadFaceDown) && IsItemHovered()) {
```
Comment (AirGuanZ) [inline imfilebrowser.h:649]: The spacing between 'if' and '(' seems inconsistent with the rest of the code, as does the missing linebreak before '{'.

**Comment by AirGuanZ** [comment]
Hi, thank you for the PR! However, there seem to be some inconsistencies in coding styles, possibly due to auto-formatting. I'd be glad to merge the PR once these are resolved.

**Comment by TheEeveeLovers** [comment]
I'll fix it! Thank you for the feedback!

**Comment by TheEeveeLovers** [comment]
OK, fixed

**Comment by AirGuanZ** [comment]
Looks good now. Thank you.

**Pull request merged**